resources/views/create.blade.php

```diff
@@ -6,40 +6,90 @@
 
     <div class="container-xl">
 
+        @if(!in_array("published_at", array_column($columns->toArray(), 'name')))
```

Comment:
Sorry I really do not understand this file and it changes. In my opinion, there should be no change to this file.